app/src/ui/diff/diff-helpers.tsx

```diff
@@ -387,3 +387,32 @@ export function getLargestLineNumber(hunks: DiffHunk[]): number {
 export function getNumberOfDigits(val: number): number {
   return (Math.log(val) * Math.LOG10E + 1) | 0
 }
+
+export function getFirstAndLastClasses(
+  row: SimplifiedDiffRow,
+  previousRow: SimplifiedDiffRow | undefined,
+  nextRow: SimplifiedDiffRow,
```

Comment:
This one could also be `undefined`
```suggestion
  nextRow: SimplifiedDiffRow | undefined,
```